CONTRIBUTING.md

```diff
@@ -72,6 +72,11 @@ by drag and dropping the file in the GitHub edition field.
 We recommend always attaching a minimal reproduction project, even if the issue
 may seem simple to reproduce manually.
 
+**Note for C# users:** If your issue is not Mono-specific, please upload a
+minimal reproduction project written in GDScript. This will make it easier for
+contributors to reproduce the issue locally as not everyone has a Mono setup
+available.
```

Comment:
```suggestion
minimal reproduction project written in GDScript (or VisualScript).
This will make it easier for contributors to reproduce the issue
locally as not everyone has a Mono setup available.
```